examples/visualization/plot_bounding_box_gallery.py

```diff
@@ -80,30 +84,29 @@ def unpackage_tfds_inputs(inputs):
     images,
     value_range=(0, 255),
     bounding_box_format="xywh",
-    y_true=boxes,
-    scale=3,
+    y_pred=boxes,
```

Comment:
why not `y_true` here anymore?

Reply:
I was just trying to show off the confidence scores in a way that would actually be used/logical